lib/src/private/mli_prv_tensor.h

```diff
@@ -17,11 +17,22 @@
 #include "mli_math_macros.h"
 #include "mli_types.h"
 #include "mli_private_types.h"
+#ifdef _ARC
+#include <arc/arc_intrinsics.h>
+#endif
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+// with a shift of 31, we cannot represent the value one. So we shift only 30
+// and an extra multiplication of 2 is done when bias is loaded.
+#define MLI_BIAS_MUL_SHIFT 30
+
+static inline uint32_t __attribute__ ((always_inline)) mli_prv_norm(int32_t val) {
+    return _norm(val);
+}
+
```

Comment:
Either add a portable (non-ARC) implementation here or remove #ifdef ARC around include arc_intrinsics.h. I assume code is not portable anyway, so probably removing ifdef is the right thing to do.

Reply:
I agree, I have removed the #ifdef ARC